libs/extensions/angular/image-banner/src/image-banner.component.html

```diff
@@ -85,6 +85,8 @@
       [attentionLevel]="backgroundBlur === 'none' ? '3' : '2'"
       [showIconOnly]="true"
       size="xs"
+      [attr.aria-label]="dismissButtonText"
+      [title]="dismissButtonText"
```

Comment:
Just talked with our UI designer, and it is preferable to leave out the title. We do not have tooltipping like this for most things throughout our solutions, and he thinks the existing affordance is enough.

```suggestion
      [attr.aria-label]="dismissButtonText"
```